### DISCUSSION TITLE
cross-origin read blocking (CORB) blocked cross-origin response when loading babel-standaloneg@6.26.0 from unpkg.com

### DISCUSSION THREAD

**Discussion by Pranay0302** [babel/babel#12359] [Q&A] [answered]
![Screenshot 2020-11-15 at 9 50 30 AM](https://user-images.githubusercontent.com/55592629/99162869-b98af980-2728-11eb-9ba7-b22ac209d9be.png)

I'm noticing this error while testing out in the localhost.

How can i resolve it? Thanks in advance.

**Comment by JLHwung** [comment, marked as answer]
https://unpkg.com/browse/babel-standalone@6.26.0/babel.min.js is an HTML file (Browse the source code)

Did you mean https://unpkg.com/babel-standalone@6.26.0/babel.min.js?

BTW, Babel Standalone 6 is long deprecated, please use https://unpkg.com/@babel/standalone@7.12.6/babel.min.js instead.

**Comment by Pranay0302** [comment]
> Did you mean https://unpkg.com/babel-standalone@6.26.0/babel.min.js?

Yep, clarified. Thanks
